db/migrate/20251022160053_change_invitation_from_id_foreign_key.rb

```
@@ -1,9 +1,14 @@
 class ChangeInvitationFromIdForeignKey < ActiveRecord::Migration[7.0]
   def change
-    # Remove old foreign key to participants
-    remove_foreign_key :invitations, column: :from_id
+    change_column :invitations, :from_id, :bigint
 
+    # Remove old foreign key to participants
+    if foreign_key_exists?(:invitations, column: :from_id)
```

RuboCop (1.86.1)

[convention] Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.

(Style/IfUnlessModifier)